src/components/seo.js

```diff
@@ -25,32 +25,40 @@ function SEO({ description, lang, meta, keywords, title }) {
       meta={[
         {
           name: `description`,
-          content: metaDescription,
+          content: 'A place for Q&A and free mentorship',
+        },
+        {
+          property: `og:title`,
+          content: 'Mentor Night | Free Code Camp',
+        },
+        {
+          property: `og:description`,
+          content: 'A place to be coached or to coach for free along your developer journey',
+        },
+        {
+          property: `og:type`,
+          content: `website`,
+        },
+        {
+          name: `twitter:card`,
+          content: `summary`,
+        },
+        {
+          name: `twitter:creator`,
+          content: '@freeCodeCamp,'
```

Comment:
If we have a Twitter account for our local chapter, we should use that instead. Let's ask @itzsaga and the others.

Reply:
https://twitter.com/nashvillefcc